contracts/utils/EscrowLib.sol

```diff
@@ -108,16 +115,19 @@ contract Escrow is IEscrow {
     * @dev calculate the USD value of all the collateral stored
     * @return - the collateral's USD value in 8 decimals
     */
-    function _getCollateralValue() internal returns (uint256) {
+    function _getCollateralValue(EscrowState storage self)
+        internal
```

Comment:
can you make sure all Library functions are external or public. No internal library functions, we are trying to reduce contract size so dont want them included in contract bytecode instead of being an a call to library.